etc/httpd/global.conf

```diff
@@ -128,8 +128,8 @@ RewriteRule ^/ja/firefox/organizations/?$ http://www.mozilla.jp/business/downloa
 # bug 1138280
 RewriteRule ^/ja/(firefox|thunderbird)/(beta/)?notes/ http://mozilla.jp/$1$2/notes/ [L,R=301]
 RewriteRule ^/ja/mobile/(beta/?)notes/ http://mozilla.jp/android/$1$2/notes/ [L,R=301]
-RewriteRule ^/ja/(firefox|thunderbird)/(\d+\.\d+(\.\d+)*)(beta)?/releasenotes(/?|/.+)$ http://mozilla.jp/firefox/$1/$2$3/releasenotes/ [L,R=301]
-RewriteRule ^/ja/mobile/(\d+\.\d+(\.\d+)*)(beta)?/releasenotes(/?|/.+)$ http://mozillla.jp/firefox/android/$1$2/releasenotes/ [L,R=301]
+RewriteRule ^/ja/(firefox|thunderbird)/((android/)?\d+\.\d+(\.\d+)*(beta)?)/releasenotes(/?|/.+)$ http://mozilla.jp/$1/$2/releasenotes/ [L,R=301]
+RewriteRule ^/ja/mobile/(\d+\.\d+(\.\d+)*(beta)?)/releasenotes(/?|/.+)$ http://mozillla.jp/firefox/android/$1$2/releasenotes/ [L,R=301]
```

Comment:
mozilla.jp currently redirects to www.mozilla.jp, so we should probably save a redirect and change the targets of all of these redirects to www.mozilla.jp